docs/install/operator/configuring-eventing-cr.md

```diff
@@ -243,3 +244,29 @@ metadata:
 spec:
   defaultBrokerClass: MTChannelBasedBroker
 ```
+
+## System Resource Settings
+
+The operator custom resource allows you allows you to configure system resources for the Knative system containers.
+Requests and limits can be configured for the following containers: `eventing-controller`, `eventing-webhook`, `imc-controller`, `imc-dispatcher`, `mt-broker-ingress`, `mt-broker-ingress` and `mt-broker-controller`.
```

Comment:
```suggestion
Requests and limits can be configured for the following containers:

- eventing-controller
- eventing-webhook
- imc-controller
- imc-dispatcher
- mt-broker-ingress
- mt-broker-ingress
- mt-broker-controller
```

Reply:
If the requests / limits are set in KnativeEventing, are they automatically applied for all of these containers, or how does that work? Do these need to be configured individually and if so do we make it clear how to do that somewhere?